server/server.js

```diff
@@ -34,6 +34,42 @@ const pool = new Pool({
 })
 
 // Express routes
+app.get("/api/guinea_pigs", (req, res) =>  {
+  pool
+  .query("SELECT * FROM adoptable_pets WHERE pet_type_id = 1")
+  .then(result =>  {
+    return res.json(result.rows)
+  })
+  .catch(error =>  {
+    console.log(error)
+  })
+})
+
+app.get("/api/reptiles", (req, res)  =>  {
+  pool
+  .query("SELECT * FROM adoptable_pets WHERE pet_type_id = 2")
+  .then(result => {
+    return res.json(result.rows)
+  })
+  .catch(error => {
+    console.log(error)
+  })
+})
+
```

Comment:
where the only difference between the /reptiles and /guinea_pigs are your pet_type_id I would suggest you combine this into one app.get and use a dynamic route
`app.get("app/v1/:pet_type")` from there you can build your query using the req 
If you restructure your query so that the where clause searches for pet_type_name instead then it'll be much simpler.